db/db.js

```diff
@@ -480,10 +480,10 @@ async function addLocationData(deviceid, phonenumber, detection_min, detection_m
   try {
       await pool.query("INSERT INTO locations(deviceid, phonenumber, detectionzone_min, detectionzone_max, sensitivity, noisemap, led, rpm_threshold, still_threshold, duration_threshold, mov_threshold, locationid) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12)", 
           [deviceid, phonenumber, detection_min, detection_max, sensitivity, noisemap, led, location]);
-      console.log("New location inserted to Database");
+      helpers.log("New location inserted to Database");
   }
   catch(e) {
-      console.log(`Error running the addLocationData query: ${e}`);
+      helpers.log(`Error running the addLocationData query: ${e}`);
```

Comment:
Thanks for updating all the console logs with the custom helper function - I'll make sure to stick to helpers going forward.

Reply:
Thanks!

Reply:
I don't know if you noticed, but the `helpers.log` version also pre-pends the timestamp, so hopefully that will be helpful when looking through the logs.